lib/matplotlib/testing/widgets.py

```diff
@@ -0,0 +1,57 @@
+"""
+========================
+Widget testing utilities
+========================
+Functions that are useful for testing widgets.
+See also matplotlib.tests.test_widgets
+"""
+import matplotlib.pyplot as plt
+from unittest import mock
+
+
+def get_ax():
+    """Creates plot and returns its axes"""
+    fig, ax = plt.subplots(1, 1)
+    ax.plot([0, 200], [0, 200])
+    ax.set_aspect(1.0)
+    ax.figure.canvas.draw()
+    return ax
+
+
+def do_event(tool, etype, button=1, xdata=0, ydata=0, key=None, step=1):
+    """
+    Trigger an event
+
+    Parameters
+    ----------
+    tool : a matplotlib.widgets.RectangleSelector instance
```

Comment:
The type is the type, not anything more.
```suggestion
    tool : matplotlib.widgets.RectangleSelector
```